docs/Queries.md

```diff
@@ -63,6 +66,15 @@ GET requests) or the request body (for POST requests).
 An array of feature flag (Runtime Flag) names that should be returned
 alongside the main request.
 
+### `meta`
+
+#### `method`
+
+You can force the query to be sent with a particular HTTP method by specifying
+it here as a _lowercase_ string: `get`, `post`, `delete`, or `patch`. Note that
+you should never try to make a request that contains multiple queries with
+different `method`s.
```

Comment:
Also, you can always generate multiple `API_REQUEST` actions if you _really_ want to do simultaneous requests with different methods - they won't be batched, but at most you would be doing 1-request-per-method from the browser